change/@fluentui-react-infolabel-75e65619-149e-4b64-9ec9-06efe295fc27.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33253&commitId=ad02743d6b909413a259ed25fe149c0ef2447432&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|10| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|4068| Changed|
|Drawer.overlay drawer full.chromium.png|2898| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The failing tests are unrelated to the changes in this PR.